### ISSUE TITLE
Replace anchors without href with buttons

### ISSUE THREAD

**Issue by cagataycivici** [primefaces/primereact#683]
Fix the warnings caused by the lint;

> The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid
